py/sentry_streams/examples/billing.py

```diff
@@ -0,0 +1,108 @@
+import json
+from typing import Optional
+
+from sentry_streams.examples.billing_buffer import PendingBuffer
+from sentry_streams.pipeline import (
+    KafkaSource,
+    Map,
+    Pipeline,
+    Reduce,
+)
+from sentry_streams.user_functions.function_template import (
+    Accumulator,
+    AggregationBackend,
+)
+from sentry_streams.window import TumblingWindow
+
+Outcome = dict[str, str]
+
+
+def build_outcome(value: str) -> Outcome:
+
+    d: Outcome = json.loads(value)
+
+    return d
+
+
+class OutcomesBackend(AggregationBackend[PendingBuffer]):
+
+    def __init__(self) -> None:
+        self.storage_map: dict[str, int] = {"state": 0, "data_cat": 0}
+
+    def flush_aggregate(self, aggregate: PendingBuffer) -> None:
+        """
+        Simply overrides the current values in the storage.
+        """
+        self.storage_map["state"] = aggregate.get_key("state")
+        self.storage_map["data_cat"] = aggregate.get_key("data_cat")
```

Comment:
If the window aggregation is also flushing to 'persistent storage' what would be in the `reduce` stream? Would it be aggregates that were flushed?